### ISSUE TITLE
Inconsistencies on the number shown in the Executive Summary slides vs the ones highlighted per focus area

### ISSUE THREAD

**Issue by novenera** [Azure/WellArchitected-Tools#46]
Hi,

I'm having challenges trying to get my head around the recommendation numbers shown in the Executive Summary slides versus the one shown per area slide.

For example, in the Reliability Pillar section, I'm seeing the following numbers in the Executive Summary:

![image](https://user-images.githubusercontent.com/16081468/218581634-8f733b55-babd-4efe-b89b-be6ad52ad250.png)

But when I go to the Application Design focus area slide, it shows 5/10 items instead of 8/10. 

![image](https://user-images.githubusercontent.com/16081468/218581740-c75d4519-10f9-4063-92b1-df9dd1efc757.png)

I cross-checked the numbers in the CSV file.  Assuming that the High items are the ones that need to be highlighted, I see that there are 8 items:

![image](https://user-images.githubusercontent.com/16081468/218583167-a90313c3-6c17-41d4-8135-60f1ca41ca30.png)

Is there anything I may have missed?

**Comment by TristankMS** [comment]
@novenera You're right, regression crept in with a recent update. Working on it now.
